catalog/opensearch/catalog-opensearch-endpoint/pom.xml

```diff
@@ -141,6 +141,7 @@
                         <Embed-Dependency>common-system</Embed-Dependency>
                         <Import-Package>
                             META-INF.cxf,
+                            javax.ws.rs.*,
```

Comment:
did you have to add this? just wondering if maven-bundle-plugin would have figured this out on its own.